components/org.wso2.carbon.identity.api.server.organization.user.sharing.management/org.wso2.carbon.identity.api.server.organization.user.sharing.management.v1/src/gen/java/org/wso2/carbon/identity/api/server/organization/user/sharing/management/v1/factories/UsersApiServiceFactory.java

```diff
@@ -23,15 +23,10 @@
 
 public class UsersApiServiceFactory {
 
-   private static final UsersApiService SERVICE = new UsersApiServiceImpl();
+   private final static UsersApiService service = new UsersApiServiceImpl();
 
-   /**
-    * Get UsersApiService.
-    *
-    * @return UsersApiService.
-    */
-   public static UsersApiService getUsersApi() {
-
-      return SERVICE;
+   public static UsersApiService getUsersApi()
+   {
```

Comment:
formatting seems off. shouldn't the opening curly brace on the same line of the method signature?